Jenkinsfile

```
@@ -58,35 +58,32 @@ pipeline {
                       "**/*.zip,**/*.tar.gz")
 
         }
-        
 
         script {
-          List labels = [];
           if (env.BRANCH_NAME == cambpmDefaultBranch()) {
             // CE master triggers EE master
             // otherwise CE PR branch triggers EE PR branch
             eeBranch = "cambpm-ee-main/pipeline-master"
           } else {
-            labels = pullRequest.labels
             eeBranch = params.EE_BRANCH_NAME
           }
 
           if (cambpmWithLabels('webapp-integration','all-as','h2','websphere','weblogic','jbosseap','run','spring-boot','authorizations')) {
-            cambpmTriggerDownstream("cambpm-ee/" + eeBranch, labels, true, true)
+            cambpmTriggerDownstream("cambpm-ee/" + eeBranch, true, true, true)
           }
 
           if (cambpmWithLabels('all-db','cockroachdb','authorizations')) {
-            cambpmTriggerDownstream("cambpm-ce/cambpm-sidetrack/${env.BRANCH_NAME}", labels)
+            cambpmTriggerDownstream("cambpm-ce/cambpm-sidetrack/${env.BRANCH_NAME}")
           }
 
           if (cambpmWithLabels('daily','default-build','rolling-update','migration','all-db','h2','db2','mysql','oracle','mariadb','sqlserver','postgresql','cockroachdb')) {
-            cambpmTriggerDownstream("cambpm-ce/cambpm-daily/${env.BRANCH_NAME}", labels)
+            cambpmTriggerDownstream("cambpm-ce/cambpm-daily/${env.BRANCH_NAME}")
```

Shouldn't `withLabel` be `true` here as well?
```suggestion
            cambpmTriggerDownstream("cambpm-ce/cambpm-sidetrack/${env.BRANCH_NAME}", true)
```